middleware/headers.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+import type { MiddlewareHandlerContext } from "$fresh/server.ts";
+
+export async function hardenHeaders(
```

Comment:
We can move this to a plugin. I want to remove the `/middleware` folder in favour of the `/plugins` folder. The plugin can have the name `security-headers`.

Reply:
I was using that approach because I wanted to apply that middleware to all routes. I want to avoid adding all the routes manually as in this snippet:

```ts
// ... imports
const middleware = { handler: hardenHeaders };
export default {
  name: "security-headers",
  middlewares: [
    {
      path: "/",
      middleware,
    },
    // ... add routes manually here
  ],
} as Plugin<State>;
```

Is there an alternative to this?

Reply:
Yep, that's the right way to do it 👍🏾